include/material/newtonian.tcc

```diff
@@ -64,24 +84,37 @@ Eigen::Matrix<double, 6, 1> mpm::Newtonian<3>::compute_stress(
     const Vector6d& stress, const Vector6d& dstrain, const ParticleBase<3>* ptr,
     mpm::dense_map* state_vars) {
 
+  // Get strain rate
+  const auto& strain_rate = ptr->strain_rate();
+
+  // Deviatoric stress component
+  Eigen::Matrix<double, 6, 1> pstress;
+  pstress(0) = 2. * dynamic_viscosity_ * strain_rate(0);
```

Comment:
Could we combine this line of `pstress` with the later line in line# 108, so we can do it once

Reply:
Good idea!